estate/models/estate_property_tag.py

```diff
@@ -0,0 +1,7 @@
+from odoo import models, fields
+
+class EstatePropertyTag(models.Model):
```

Comment:
```suggestion


class EstatePropertyTag(models.Model):
```
we normally add 2 empty lines before defining class, check and update it in other files too
:) refer codebase